aspnetcore/tutorials/first-web-api.md

```diff
@@ -201,6 +201,11 @@ In contrast, the `GetById` method returns the more general `IActionResult` type,
 
 In Visual Studio, press CTRL+F5 to launch the app. Visual Studio launches a browser and navigates to `http://localhost:port/api/values`, where *port* is a randomly chosen port number. If you're using Chrome, Edge or Firefox, the data will be displayed. If you're using IE, IE will prompt to you open or save the *values.json* file. Navigate to the `Todo` controller we just created `http://localhost:port/api/todo`.
 
+> [!NOTE]
+> If you are attempting to use IIS Express (default) and the application does not display in your browser, check to make sure that your Main function in Program.cs includes ```csharp .UseIISIntegration()```
+
+[!code-csharp[Main](first-web-api/sample/src/TodoApi/Program.cs)]
```

Comment:
Let's close this down just to the Main method and highlight the row, so modify the *Program.cs* file in the sample and place a "snippet1" around the Main method. Look at the following example from the ToDoController. The snippets are created using `#region snippet_name` and `#endregion`: https://github.com/aspnet/Docs/blob/master/aspnetcore/tutorials/first-web-api/sample/src/TodoApi/Controllers/TodoController.cs#L17-L34

... then refer to the snippet and highlight the line like I show below. I'm using "snippet1" here, but you can name it something more explicit if you like. The highlight counting starts with the first line in the snippet, so the `.UseIISIntegration()` line should be line #6 ...
```
[!code-csharp[Main](first-web-api/sample/src/TodoApi/Program.cs?name=snippet1&highlight=6)]
```